config/default.yml

```diff
@@ -23,6 +23,9 @@ RSpec/ExampleWording:
     not: does not
   IgnoredWords: []
 
+RSpec/BeforeAfterAll:
+  Description: 'Avoid the use of before/after(:all) as they are not rolled back and may lead to database state leaking between examples.'
```

Comment:
I think it would be helpful to cover both situations, Rails and non-Rails, since using Rails is such a common use case. For example:

> Avoid the use of before/after(:all) as it prevents examples from being fully isolated, meaning state may leak between tests. (For Rails apps, this means the database may not be cleared between tests').